Microsoft.Toolkit.Uwp.Notifications/Toasts/Compat/Desktop/Native/CAppResolver.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if WIN32
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Toolkit.Uwp.Notifications
+{
+    [ComImport]
+    [Guid("660b90c8-73a9-4b58-8cae-355b7f55341b")]
```

Comment:
Do these Guids map to anything particular or are we just pinning?

Reply:
Yes, they map to the specific C++ APIs that get called (those native APIs have these GUID identifiers set), since I believe C# looks up what specifically to call based on that GUID.